src/azul/service/elasticsearch_service.py

```diff
@@ -587,6 +587,15 @@ def transform_request(self,
                 es_response = es_search.execute(ignore_cache=True)
             except elasticsearch.NotFoundError as e:
                 raise IndexNotFoundError(e.info["error"]["index"])
+            except elasticsearch.RequestError as e:
+                if one(e.info['error']['root_cause'])['reason'].startswith('No mapping found for'):
+                    final_response = FileSearchResponse(hits={},
```

Comment:
I would like to add a round-trip to ES here, the `_count` API to confirm that the index exists but is indeed empty.

Reply:
https://www.elastic.co/guide/en/elasticsearch/reference/6.8/search-count.html